data/db/mil-tamron.xml

```diff
@@ -1,6 +1,109 @@
 <!DOCTYPE lensdatabase SYSTEM "lensfun-database.dtd">
 <lensdatabase version="2">
 
+    <lens>
+        <maker>Tamron</maker>
+        <model>E 17-70mm F2.8 B070</model>
+        <mount>Sony E</mount>
+        <cropfactor>1.5</cropfactor>
+        <type>rectilinear</type>
+        <calibration>
+            <distortion model="ptlens" focal="17" a="0.02013" b="-0.0584" c="0"/>
+            <distortion model="ptlens" focal="24" a="0.01469" b="-0.03005" c="0.01989"/>
+            <distortion model="ptlens" focal="31" a="0.00962" b="-0.01067" c="0.00847"/>
+            <distortion model="ptlens" focal="43" a="0.00598" b="0.00046" c="0.00312"/>
+            <distortion model="ptlens" focal="59" a="0.00288" b="0.00548" c="0.00614"/>
+            <distortion model="ptlens" focal="70" a="0.00217" b="0.01183" c="-0.00867"/>
+            <tca model="poly3" focal="17" vr="1.0000980" vb="1.0002878"/>
+            <tca model="poly3" focal="17" vr="1.0000980" vb="1.0002907"/>
+            <tca model="poly3" focal="17" vr="1.0001041" vb="1.0002558"/>
+            <tca model="poly3" focal="17" vr="1.0001081" vb="1.0002728"/>
+            <tca model="poly3" focal="17" vr="1.0001090" vb="1.0002675"/>
+            <tca model="poly3" focal="17" vr="1.0001119" vb="1.0002827"/>
+            <tca model="poly3" focal="17" vr="1.0001127" vb="1.0002940"/>
+            <tca model="poly3" focal="17" vr="1.0001170" vb="1.0002900"/>
+            <tca model="poly3" focal="17" vr="1.0001349" vb="1.0002532"/>
+            <tca model="poly3" focal="17" vr="1.0001422" vb="1.0002700"/>
+            <tca model="poly3" focal="17" vr="1.0001451" vb="1.0002370"/>
+            <tca model="poly3" focal="17" vr="1.0001536" vb="1.0002586"/>
```

Comment:
You seem to have lots of entries for TCA at each focal length. I think these need to be reduced to a single entry.

Reply:
Okay, I'll fix that.

Reply:
I was hoping this would fix the autobuilds but it hasn't. If you can fix them then I'll merge this, or hopefully I'll have a chance to have a look at it later in the week.